docs/testnet/tutorial.md

```diff
@@ -0,0 +1,158 @@
+# Getting Started
+In this tutorial, you build your own Obscuro dApp from the start. This is an excellent way to experience a typical development process as you learn how Obscuro dApps are built, design concepts, tools and terminology.
+
+# Let's go
+This 'Guessing Game' tutorial provides an introduction to the fundamentals of Obscuro and shows you how to:
+- Set up your local Ethereum development environment for Obscuro.
+- Use Obscuro concepts to develop a dApp
+
+What is the *Guessing Game*?
+
+The Guessing Game is a dApp that you will build that demonstrates a basic Obscuro use case, a simple number guessing game. The contract generates a random secret number, which is never revealed to anyone, including node operators, developers or end-user because of the privacy provided by Obscuro. The goal of the game is to guess this number, and each time an attempt is made, an entrance fee of 1 token is paid. If a user correctly guesses the number, the contract will pay out all of the accumulated entrance fees to them and reset itself with a new random number.
```

Comment:
I think you can start strong here and say, "The Guessing Game is an example of a dApp that can only be built on Obscuro, and not on native Ethereum"